documentation/asciidoc/topics/proc_configuring_backup_sites.adoc

```diff
@@ -0,0 +1,65 @@
+[id='configuring_backup_sites-{context}']
+= Configuring {brandname} Sites
+
+.Procedure
+. Create custom resources for each site.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml_annotated/cr_backup_site.yaml[]
+----
++
+<1> specifies the DataGrid service. {brandname} supports cross-site replication with the DataGrid service only.
+<2> names the site where the {brandname} cluster runs, the local site.
+//Community
+ifndef::productized[]
+<3> defines the externally exposed service.
+* Use `NodePort` for local clusters on the same network. For example, if you use Minikube for testing purposes.
+* Use `LoadBalancer` for independent {openshiftshort} clusters.
+endif::productized[]
+//Product
+ifdef::productized[]
+<3> specifies `LoadBalancer` as the service that exposes the {openshiftshort} cluster to externally network traffic.
+endif::productized[]
+<4> names the site to which the {brandname} cluster replicates data, the backup site.
+//Community
+ifndef::productized[]
+<5> specifies the URL to access the backup site.
+* Use `xsite://` if the backup site has a static hostname and port.
+* Use `minikube://` if the backup site is a Minikube instance.
```

Comment:
Not sure if you want to do it here or elsewhere, but for `minikube://` and `openshift://` I'd stress that the type of remote cluster, as defined in the URL scheme, has an impact on what the secret must contain. If the remote is minikube, then it expects the secret to contain ..., and if openshift then .... I think you might already mention it elsewhere, but the key is the scheme the URL uses.

Reply:
thx @galderz updated the docs.

Reply:
Where did you add that info? I cannot find it in the pushed update